src/Components/SimpleCard.js

```diff
@@ -0,0 +1 @@
+export default function SimpleCard() {}
```

Comment:
try to have no empty components. Just write something like <h1>Some text<h1> to make sure you return something

Reply:
Thanks, i was ready to start my next story, then i realize i need to do the Qa first